cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/HTableNameConverter.java

```diff
@@ -67,38 +66,33 @@ private String getBackwardCompatibleTableName(String tablePrefix, TableId tableI
    */
   protected String getHBaseTableName(String tablePrefix, TableId tableId) {
```

Comment:
nit: since we have a ``fromHBaseTableName``, should this be called ``toHBaseTableName``?